tmt/steps/provision/__init__.py

```diff
@@ -778,7 +778,8 @@ def _random_name(self, prefix: str = '', length: int = 16) -> str:
         # Append at least 5 random characters
         min_random_part = max(5, length - len(prefix))
         name = prefix + ''.join(
-            random.choices(string.ascii_letters, k=min_random_part))
+            # S311: Use of pseudo-random generator is adequate for tmt run id
+            random.choices(string.ascii_letters, k=min_random_part))  # noqa: S311
```

Comment:
@martinhoyer maybe we could just use the secrets module? The no noqa magic needed

Note that this will require `import secrets` at the start

```suggestion
            ''.join(secrets.choice(string.ascii_letters) for _ in range(min_random_part))
```

Reply:
Thanks. Interestingly it even looks like it's not more expensive.